draft-ietf-quic-ack-frequency.md

```diff
@@ -341,6 +341,13 @@ flight at any given time, this extension does not prohibit having more than one
 in flight. Generally, when using `max_ack_delay` for PTO computations, endpoints
 MUST use the maximum of the current value and all those in flight.
 
+When the number of in-flight ack-eliciting packets is larger than the
+ACK-Eliciting Threshold, an endpoint can expect that the peer will not need to
+wait for its `max_ack_delay` period before sending an acknowledgement. In such
+cases, the endpoint MAY therefore exclude the peer's 'max_ack_delay' from its PTO
+calculation.  If `ignore_order` is enabled and packet(s) are lost, this
+optimization can cause premature PTOs, so requires extra caution.
```

Comment:
```suggestion
calculation. Note that this optimization requires some care in implementation, since
it can cause premature PTOs under packet loss when `ignore_order` is enabled.
```